src/card.css

```diff
@@ -0,0 +1,78 @@
+.card {
+  background-color: var(--white-color);
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  cursor: pointer;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: var(--font-size-s);
+  line-height: var(--line-height-l);
+  font-weight: 500;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__product-code {
+  color: #616070;
+  line-height: var(--line-height-s);
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-quantity {
+  line-height: var(--line-height-s);
+}
```

Comment:
check styles on this elem in figma
color not good

Reply:
Yes, I forgot about the color. Thanks